Divine/CLI/CommandLineArguments.cs

```diff
@@ -0,0 +1,310 @@
+            ValueOptional = false,
+            Optional = true
+        )]
+        public string[] Options;
+
+        [ValueArgument(typeof(string), "conform-path", 
+            Description = "Set conform to original path", 
+            DefaultValue = null,
+            ValueOptional = false,
+            Optional = true
+        )]
+        public string ConformPath;
+
+        [EnumeratedValueArgument(typeof(string), 'p', "package-version",
+            Description = "Set package version",
+            DefaultValue = "v13",
+            AllowedValues = "v7;v9;v10;v13",
+            ValueOptional = false,
+            Optional = true
+        )]
+        public string PackageVersion;
+
+        [EnumeratedValueArgument(typeof(string), 'c', "compression-method",
+            Description = "Set compression method",
+            DefaultValue = "lz4hc",
+            AllowedValues = "zlib;zlibfast;lz4;lz4hc;none",
+            ValueOptional = false,
+            Optional = true
+        )]
+        public string CompressionMethod;
+
+        public static LogLevel GetLogLevelByString(string optionLogLevel)
+        {
+            LogLevel logLevel;
+
+            switch (optionLogLevel)
+            {
+                case "silent":
+                    logLevel = Enums.LogLevel.SILENT;
+                    break;
+                case "info":
+                    logLevel = Enums.LogLevel.INFO;
+                    break;
+                case "warn":
+                    logLevel = Enums.LogLevel.WARN;
+                    break;
+                case "error":
+                    logLevel = Enums.LogLevel.ERROR;
+                    break;
+                case "fatal":
+                    logLevel = Enums.LogLevel.FATAL;
+                    break;
+                case "debug":
+                    logLevel = Enums.LogLevel.DEBUG;
+                    break;
+                default:
+                    logLevel = Enums.LogLevel.INFO;
+                    break;
+            }
+
+            return logLevel;
+        }
+
+        public static Game GetGameByString(string optionGame)
+        {
+            Game game = Enums.Game.DivinityOriginalSin2;
+
+            switch (optionGame)
+            {
+                case "dos":
+                    game = Enums.Game.DivinityOriginalSin;
+                    break;
+                case "dosee":
+                    game = Enums.Game.DivinityOriginalSinEE;
+                    break;
+                case "dos2":
+                    game = Enums.Game.DivinityOriginalSin2;
+                    break;
+            }
+
+            return game;
+        }
+
+        public static int GetFileVersionByGame(Game divinityGame)
+        {
+            return divinityGame == Enums.Game.DivinityOriginalSin2
+                ? (int)LSLib.LS.LSF.FileVersion.VerExtendedNodes

```

Comment:
Note to self: FileVersion should really be an enum. I have no idea why it is a struct with const members ...